src/components/header.js

```diff
@@ -125,9 +128,11 @@ const NavLink = styled(Link)`
 const CreateLink = styled.div`
   cursor: pointer;
   text-decoration: none;
-  font-family: 'Red Hat Display', sans-serif;
+  font-family: 'Red Hat Text', sans serif;
```

Comment:
You can also reference this value as `${theme.fonts.body}` or `${theme.fonts.heading}` if you need Red Hat Display

Same goes for `line-height`, `font-weight`, etc.

If you catch yourself doing many of these - it pays to look at the defined variants and apply them directly when returning a new component - i.e. as sx={{variant: 'text.default'}}

Local styles should be mainly used for positioning, spacing and if you need to apply CSS to children (I am sadly guilty of using anti-patterns to theme-ui myself but one of my goals is to reduce visual style away from styled-components to variants. 

This approach is great for white labeling, but beware - the theme files need to have the same variants present, otherwise things will break between different meta-projects. That is also the reason why it makes more sense IMO to use only one theme file and not multiple ones within one repository.

Reply:
Yes! I'm trying to keep it as clean as possible. There's some mix because some old code was like that. The refactor will happen for sure